pkg/options/collector.go

```diff
@@ -53,5 +53,6 @@ var (
 		"storageclasses":                  struct{}{},
 		"validatingwebhookconfigurations": struct{}{},
 		"volumeattachments":               struct{}{},
+		"leases":                          struct{}{},
```

Comment:
This needs to be in lexical order too please.

Reply:
fixed